tools/otci/otci/otci.py

```diff
@@ -947,6 +947,11 @@ def __parse_srp_server_service_txt(self, txt: str) -> Dict[str, Union[bytes, boo
 
         return txt_dict
 
+    def srp_server_get_lease(self):
```

Comment:
```suggestion
    def srp_server_get_lease(self) -> Tuple[int, int, int, int]:
```

We tend to use typing in OTCI.